packages/core/src/scss/components/checkout/customer/_customer.scss

```diff
@@ -49,9 +49,15 @@
 }
 
 .stripeCustomerEmail-button {
```

Comment:
Just thinking should we have a separate file for stripe styles? I mention this because in future we would like to move all stripe related code to it's own package which means we probably also look at moving the styles across as well.

Reply:
Hey @animesh1987, Is this approach ok for you?

please let me know your thoughts